src/odemis/driver/test/xt_client_test.py

```diff
@@ -35,7 +35,9 @@
 from odemis.model import ProgressiveFuture, NotSettableError
 from odemis.util import test
 
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(level=logging.DEBUG,
```

Comment:
Did you check if the test file scanner_test.py with the composited scanner/detector still passes? It might influence that test case

Reply:
just checked, and that still passes